### PR TITLE
daemon/graphdriver fix lint errors/warnings

### PR DESCRIPTION
**Pull request by brahmaroutu**
Addresses #14756
Signed-off-by: Srini Brahmaroutu srbrahma@us.ibm.com

### CODE REVIEW

**Comment by jeffdm** [comment]
There's a couple warnings from driver_windows & driver_unsupported. From running `make validate`:

```
Errors from golint:
daemon/graphdriver/driver_unsupported.go:12:1: exported function GetFSMagic should have comment or be unexported
daemon/graphdriver/driver_windows.go:8:6: exported type WindowsGraphDriver should have comment or be unexported
daemon/graphdriver/driver_windows.go:10:15: interface method parameter sourceId should be sourceID
daemon/graphdriver/driver_windows.go:26:1: exported function GetFSMagic should have comment or be unexported

```

File: daemon/graphdriver/driver.go [hunk 1]
```diff
@@ -89,6 +96,7 @@ func init() {
 	drivers = make(map[string]InitFunc)
 }
 
+// Register register a InitFunc for the driver.
```
Comment (jeffdm) [inline daemon/graphdriver/driver.go]: maybe replace `register a` with `registers an`

File: daemon/graphdriver/driver.go [hunk 2]
```diff
@@ -98,6 +106,7 @@ func Register(name string, initFunc InitFunc) error {
 	return nil
 }
 
+// GetDriver initializes and return the registered driver
```
Comment (jeffdm) [inline daemon/graphdriver/driver.go]: maybe replace `return` with `returns`

File: daemon/graphdriver/driver.go [hunk 3]
```diff
@@ -106,6 +115,7 @@ func GetDriver(name, home string, options []string) (Driver, error) {
 	return nil, ErrNotSupported
 }
 
+// New creates the driver and initializes it and the specified root.
```
Comment (jeffdm) [inline daemon/graphdriver/driver.go]: initializes it *at* the specified root?

**Comment by brahmaroutu** [comment]
@jeffdm Please review the revised comments.

**Comment by icecrime** [comment]
Needs rebase.

**Comment by brahmaroutu** [comment]
@icecrime rebased.

**Comment by calavera** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Pull request merged**